### ISSUE TITLE
Pixels are choppy and intersect when magnet is on top

### ISSUE THREAD

**Issue by brooklynlash** [phetsims/faradays-law#194]
Test device
Lenovo ThinkPad

Operating System
Windows 10

Browser
Chrome

Problem description
Pixels look a bit choppy when moving a magnet on top of the coil
Also occurs in published. 

Visuals
![magnet](https://user-images.githubusercontent.com/72031637/97630888-cd7aef80-19f5-11eb-969c-86eeeaa31d95.PNG)
![magnet2](https://user-images.githubusercontent.com/72031637/97630899-d075e000-19f5-11eb-8565-f1567213b11f.PNG)

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Faraday's Law‬
URL: https://phet-dev.colorado.edu/html/faradays-law/1.4.0-dev.65/phet/faradays-law_en_phet.html?
Version: 1.4.0-dev.65 2020-10-27 23:56:15 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.111 Safari/537.36
Language: en-US
Window: 1707x818
Pixel Ratio: 2.25/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4095
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

**Comment by brooklynlash** [comment]
This is for https://github.com/phetsims/QA/issues/568

**Comment by KatieWoe** [comment]
I think this is in published. so marking as deferred. @brooklynlash if it isn't in published, let me know.

**Comment by jbphet** [comment]
Undeferring, since I think I can fix this fairly easily, so we might as well get it into the upcoming release.

**Comment by jbphet** [comment]
I've made some adjustments to the code that defines the "restricted areas" where the magnet is blocked by the coils.  The results looks significantly better to my eye, though there is still a small amount of overlap on the right top of the coil on my machine.  I am trying to keep the boundary information in integer values since it makes the code a bit cleaner, so this is probably as good as we can get.

I've tweaked a number of the values for the restricted areas.  @KatieWoe - please have QA test on master and verify that this is better and let me know if you (or whoever tests it) feel like this sufficiently addresses the issue.  If you feel like a designer needs to approve it (I don't, but I'll leave that decision to you), please assign to @arouinfar.

**Comment by KatieWoe** [comment]
Checked on Win 10 Chrome on master (12/2/2020). Looks much better. Can still see some edges if you get in there and squint, (and the resolution on this device is pretty good), but you're unlikely to notice in regular use. The field lines still go through the coils, but I'm not sure if anything can or should be done about that instance.
<img width="173" alt="muchbetter" src="https://user-images.githubusercontent.com/41024075/100921274-891dbc00-3499-11eb-8cd8-7c79caa45532.png">
<img width="146" alt="linethrough" src="https://user-images.githubusercontent.com/41024075/100921420-caae6700-3499-11eb-8023-c7d94dce74ee.png">

**Comment by jbphet** [comment]
> The field lines still go through the coils, but I'm not sure if anything can or should be done about that instance.

That's intentional - they have to go somewhere.

Since this one is pretty subjective and we agree that it's an improvement, I'm going to go ahead and close it instead of having it tested in the upcoming RC.
